app/assets/javascripts/es6-components.js

```diff
@@ -0,0 +1,14 @@
+// These modules from govuk_publishing_components
+// depend on govuk-frontend modules. govuk-frontend
+// now targets browsers that support `type="module"`.
+//
+// To gracefully prevent execution of these scripts
+// on browsers that don't support ES6, this script
+// should be included in a `type="module"` script tag
+// which will ensure they are never loaded.
+
+//= require govuk_publishing_components/components/skip-link
+//= require govuk_publishing_components/components/layout-header
+//= require govuk_publishing_components/components/error-summary
+//= require govuk_publishing_components/components/button
+//= require govuk_publishing_components/components/radio
```

Comment:
Could these be sorted into alphabetical order?

Reply:
Done 👍